test/splitter.js

```diff
@@ -97,34 +89,61 @@ contract('Splitter', async accounts => {
         checkEventNotEmitted();
     });
 
+    it("splitDeposit is pausable", async () => {
+
+        splitter.pause({from: contractOwner});
```

Comment:
You ought to test also from other accounts.
And unless you `await` it, it will not work. Or rather, it will work only with Ganache because it mines so fast.

Reply:
Noted. Thank you. I have added a test to check that non-owner account calls fail when paused.